### ISSUE TITLE
PLayFixtureModule not working 

### ISSUE THREAD

**Issue by laiboonh** [scalikejdbc/hello-scalikejdbc#11]
In your example you enabled the module but did not use it. I tried using it and it does not work. No error is produced but it seems the fixture did not make a difference.
```
db.default.fixtures.dev=["companies.sql"]
```
shifted insert statements over
```
insert into company (name, url, created_at) values ('Typesafe', 'http://typesafe.com/', current_timestamp);
insert into company (name, url, created_at) values ('Oracle', 'http://www.oracle.com/', current_timestamp);
insert into company (name, url, created_at) values ('Google', 'http://www.google.com/', current_timestamp);
insert into company (name, url, created_at) values ('Microsoft', 'http://www.microsoft.com/', current_timestamp);
```

**Comment by seratch** [comment]
Thanks. We should fix this issue.

**Comment by seratch** [comment]
Fixed by #14 Thanks @SAMMY7th !
